.github/workflows/update-gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+name: Update gh-pages
+on:
+  push:
+    branches: [ main ]
```

Comment:
This should be master here, at this point? And then followup to change to main?

Reply:
It can be! I wasn't sure if we were ready for all of this to happen yet, so I was letting that happen at your leisure.

Reply:
Ah, ok! I think this was the part that I was least sure about.  So, maybe let's land this as master first, verify switching the pages source works just fine, and then I can look at switching main and master later today.